src/Neo.VM/EvaluationStack.cs

```diff
@@ -45,7 +45,7 @@ public override string ToString()
                             StackItemType.Pointer => $"({((Pointer)p).Position})",
                             StackItemType.Boolean => $"({p.GetBoolean()})",
                             StackItemType.Integer => $"({p.GetInteger()})",
-                            StackItemType.ByteString => $"(\"{p.GetString()}\")",
+                            StackItemType.ByteString => p.GetSpan().ToArray().TryGetString(out var str) ? $"(\"{str}\")" : $"(\"{Convert.ToBase64String(p.GetSpan())}\")",
```

Comment:
Mixing base64 and strings (that can be valid base64 as well) is not a good idea to me, how can I differentiate `41414141` from `000000` then?

This also is an incompatible change.

Reply:
> Mixing base64 and strings (that can be valid base64 as well) is not a good idea to me, how can I differentiate `41414141` from `000000` then?
> 
> This also is an incompatible change.

What? this is only for the ease of debug and unit test,,,, how come it can be incompatible to something? we dont print the executionstack anywhere but unit test.

Reply:
I don't know how exactly it's used in the current codebase, but it's `public`. If it's debug/test only, then maybe it's OK (can it be used by plugins/other tools?).

Reply:
yes, cause i originally added this to debug the devpack, currently nowhere in the master branch is using it.

Reply:
Doesn't matter it's in the main `Neo.VM`. If it's for tests then add it to the unit test's code.

Reply:
thought about it, but we will need it cross all unit tests, including devpack.

Reply:
Shouldn't put test in main code. It can get mixed up to easily. Especially for a new person and if it's not documented. If it ends up  being in main code. It needs to fail, unless running in a test. Just create new library for unit tests functionality that is the same for the test projects.

Reply:
will update.

Reply:
Refs. #3033 (not in any release yet, can be changed/removed). I agree with @cschuchardt88, this shouldn't be widely exposed if it's debug/test only.